charts/stable/omada-controller/values.yaml

```diff
@@ -26,15 +26,48 @@ service:
     ports:
       http:
         port: 8043
+  management:
+    enabled: false
+    ports:
+      tcp29811:
```

Comment:
Would it be possible to add descriptive names to these ports?

Reply:
FYI, 29811, 29812, 29813 are all device upgrade/management ports.